build.gradle.kts

```diff
@@ -12,28 +12,28 @@ allOpen {
 }
 
 dependencies {
-  implementation("org.springframework.boot:spring-boot-starter-data-jpa")
-  implementation("org.apache.poi:poi-ooxml:4.1.2")
   implementation("com.github.kittinunf.result:result:3.1.0")
   implementation("com.github.kittinunf.result:result-coroutines:3.1.0")
   implementation("com.beust:klaxon:5.4")
-  implementation("org.springframework.boot:spring-boot-starter-thymeleaf")
-  implementation("com.amazonaws:aws-java-sdk-s3:1.11.932")
+  implementation("com.amazonaws:aws-java-sdk-s3:1.11.940")
```

Comment:
As well as bumping some of the 3rd party dependencies I have reordered to be in alpha order.